tools/cast_live_python

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+#
+set -u -e
+
+# reduce confusion with xdotool
+setxkbmap us
+
+test ! -e $1 && echo "input file does not exist" && exit 1
+title="$(echo $(basename $1) | sed -e 's/.sh$//')"
+bashrc_file="$(dirname $0)/cast_ipython.rc"
```

Comment:
so overall this script seems to be the `cast_live` without much of changes besides a different .rc file and executing the ipython command.  So I feel that 
- the `cast_live_ipython` should merely define a few env variables (`CAST_SHELL=ipython`)
- there might be no need for cast_ipython.rc, ipython could be executed within `cast_live`: `if [ ! -z "$CAST_SHELL" ] ; then $CAST_SHELL; fi` or smth like that .  If it would really be needed (e.g. to setup ipython config) then it could just source `cast_bash.rc` and just complement with additional needed stuff

What other customization need to be done?  could probably also be channeled via environment variables.
The problem with having two largely duplicate scripts is difficulty in maintaining them.